packages/pn-landing-webapp/api/data/en.tsx

```diff
@@ -555,11 +538,11 @@ const paHorizontalNav: HorizontalNavProps = {
           <img src="static/icons/HORIZONTAL-NAV-1.svg" />
         </SvgIcon>
       ),
-      title: "Rappresenti un’impresa?",
+      title: "Do you represent a company?",
```

Comment:
I could not find any uses of neither `paHorizontalNav` nor `pfHorizontalNav`. I found the `getHorizontalNavData` in `api/index.ts`, but it seems to have no uses. Is this information included in view or a future version, or else it could be erased?

Reply:
The HorizontalNav component has been removed from the layout but it may be a temporary decision. I received its translations so I think we'd better keep it and at most put a comment right above the getHorizontalNavData method.